package.json

```diff
@@ -17,23 +24,29 @@
     "lint-staged": "^13.2.0",
     "marked": "^0.4.0",
     "prettier": "^2.8.4",
+    "publint": "^0.2.1",
     "ts-node": "^8.4.1",
+    "tsup": "^7.2.0",
     "typedoc": "^0.11.1",
     "typescript": "^5.0.4",
     "vitest": "^0.28.4"
   },
   "scripts": {
     "clean": "rm -rf ./dist && mkdir dist",
     "test": "vitest run --coverage && yarn run compile",
+    "tsup:test": "vitest run --coverage && yarn run tsup:watch",
     "test:ci": "vitest run",
     "compile": "tsc",
     "compile:dist": "tsc --project tsconfig.dist.json",
+    "tsup:watch": "tsup --watch",
     "build": "yarn run clean && yarn run build:es && yarn run build:cjs",
     "build:es": "tsc --project tsconfig.build.json --module esnext --outDir \"./dist/es\"",
     "build:cjs": "tsc --project tsconfig.build.json --module commonjs --outDir \"./dist/commonjs\"",
+    "tsup:build": "tsup",
     "prettier": "prettier . --write",
     "prettier:check": "prettier . --check",
     "lint": "eslint src/**/*.ts --fix",
+    "publint": "publint",
```

Comment:
Unless we run this in some CI pipeline there's no need to define a script for it. It's just mirrors `npx publint` (`npm run publint`)
```suggestion
```

Reply:
I could not get this package to work with code sandbox. I will try to use the build scripts that were written to execute tsup instead.